wrapper/CMakeLists.txt

```diff
@@ -88,6 +91,8 @@ target_link_libraries(NUTC-client_exe PRIVATE CURL::libcurl)
 target_link_libraries(NUTC-client_exe PRIVATE glaze::glaze)
 target_link_libraries(NUTC-client_exe PRIVATE pybind11::pybind11)
 target_link_libraries(NUTC-client_exe PRIVATE Python::Python)
+message (STATUS "PYTHON_LIBRARIES: ${PYTHON_LIBRARIES}")
```

Comment:
```suggestion

message(STATUS "PYTHON_LIBRARIES: ${PYTHON_LIBRARIES}")
```

Reply:
Removed this debug message in a commit 2secs ago